### PR TITLE
chore: make navbar reusable in templates

### PR DESCRIPTION
**Pull request by filiptammergard**

### CODE REVIEW

**Comment by LucidityDesign** [comment]
What do you think of adding a story to a group called "mocks" or "wireframe" or something like that?

**Comment by filiptammergard** [comment]
> What do you think of adding a story to a group called "mocks" or "wireframe" or something like that?

Do you mean for the navbar (and not the table view)?

**Comment by LucidityDesign** [comment]
> > What do you think of adding a story to a group called "mocks" or "wireframe" or something like that?
> 
> Do you mean for the navbar (and not the table view)?

Yes, the navbar

**Comment by filiptammergard** [comment]
Maybe something like this?

File: src/templates/components/Navbar/Navbar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, Story as StoryType } from "@storybook/react"
+import { Box } from "../../../components/layout/Box/Box"
+import { Navbar } from "./Navbar"
+
+export default {
+  title: "Templates/Wireframes/Navbar",
+  component: Navbar,
+  decorators: [
+    (Story: StoryType) => (
+      // Resetting margin from global decorator. Should be remove the global decorator, or can it be used conditionally?
```
Comment (LucidityDesign) [inline src/templates/components/Navbar/Navbar.stories.tsx:10]: Maybe you can pass an argument to the story wrapper like that?
https://dev.to/tmikeschu/controlling-global-decorators-via-args-in-storybook-41lf

Comment (filiptammergard) [reply src/templates/components/Navbar/Navbar.stories.tsx:10]: Hmm, not sure how you mean. Maybe you can push a commit to show?

Comment (LucidityDesign) [reply src/templates/components/Navbar/Navbar.stories.tsx]: Please see 5b1cb72

**Comment by filiptammergard** [comment]
Nice! Updated now to use this way in both places!

**Review by LucidityDesign — APPROVED** [review]

**Pull request merged**